src/pages/select-your-country.jsx

```diff
@@ -17,91 +18,84 @@ export default ({ data }) => {
 
   return (
     <Layout title={data.site.siteMetadata.title} seoTitle="Select Your Country">
-      <CountryContext.Consumer>
-        {({ country, setCountry, clearCountry }) => (
-          <div className="select-your-country padding">
-            <div className="header">
-              <h1>Select Your Country</h1>
-              <input
-                type="text"
-                placeholder="Filter Countries"
-                onChange={e => setSearchTerm(e.target.value.toLowerCase())}
-              />
+      <div className="select-your-country padding">
+        <div className="header">
+          <h1>Select Your Country</h1>
+          <input
+            type="text"
+            placeholder="Filter Countries"
+            onChange={e => setSearchTerm(e.target.value.toLowerCase())}
+          />
+        </div>
+        <div>
+          {country.name !== null ? (
+            <div className="country-list-item selected" key={country.code}>
+              <small>Currently selected</small>
+              <div>
+                {flag(country.code)} {country.name}
+              </div>
+              <a
+                href="#"
+                onClick={e => {
+                  e.preventDefault();
+                  clearCountry();
+                  navigate('/');
+                }}
+              >
+                Remove
+              </a>
             </div>
-            <div>
-              {country.name !== null ? (
-                <div className="country-list-item selected" key={country.code}>
-                  <small>Currently selected</small>
-                  <div>
-                    {flag(country.code)} {country.name}
+          ) : null}
+          {Object.keys(countries)
+            .filter(countryCode =>
+              countries[countryCode].toLowerCase().includes(searchTerm)
+            )
+            .map(countryCode => {
+              const countryHasContent = countriesWithContent.includes(
+                countryCode
```

Comment:
I think this might need to be `countryCode.toLowerCase()`

Reply:
Good catch! Just pushed a fix